src/test/java/io/crate/jmx/integrationtests/V570MetricsITest.java

```diff
@@ -22,12 +22,24 @@
 
 package io.crate.jmx.integrationtests;
 
-public class V440MetricsITest extends MetricsITest {
+import org.junit.Test;
 
-    private static final String URL = "https://cdn.crate.io/downloads/releases/crate-4.4.0.tar.gz";
+public class V570MetricsITest extends MetricsITest {
```

Comment:
Same here, either extend from the others or remove. Duplication for no reason is not so great.